src/pnpq/units.py

```diff
@@ -34,7 +54,7 @@ def to_mpc320_velocity(
 
     converted_velocity = (velocity_in_degrees / mpc320_max_velocity) * 100
     rounded_velocity: Quantity = (
-        int(round(converted_velocity).magnitude) * ureg.dimensionless
+        int(round(converted_velocity).magnitude) * ureg.mpc320_velocity
```

Comment:
This line is actually really funky. I think this can be simply written as
```suggestion
        round(converted_velocity.magnitude) * ureg.mpc320_velocity
```